app.rb

```diff
@@ -68,6 +68,16 @@ class Application < Sinatra::Base
 
     erb :index
   end
+  get "/requests" do
+    unless session[:user_id].nil?
+      @repo = ReservationRepository.new
```

Comment:
i would change this to reservation_repo for semantic reasons

Reply:
Will do